app/services/DR/getAllCases.js

```diff
@@ -1,12 +1,30 @@
-import { COV_CASES_SERVICE } from '../../constants/DR/baseUrls';
-import fetch from '../../helpers/Fetch';
-
-export function getAllCases(date) {
-  return fetch(`${COV_CASES_SERVICE}${date ? `?date=${date}` : '/'}`)
-    .then(({ data }) => {
-      return data;
-    })
-    .catch(err => {
-      return err.response.status;
+import Axios from 'axios';
```

Comment:
There's no need to import Axios in here because the import `import fetch from '../../helpers/Fetch'` is using axios in the background, you can check that out in `Fetch.js` file